lib/miq_automation_engine/engine/miq_ae_engine/state_var_hash.rb

```diff
@@ -33,6 +32,16 @@ def init_with(coder)
 
     private
 
+    # returns `id` of new blob
+    def generate_binary_blob
+      blob               = BinaryBlob.new
+      blob.resource_type = "StateVarHash"
```

Comment:
I don't mind the method extraction, but I think this PR should still remove the resource_type, since it's not a "real" value for that column, and isn't necessary to the way the purger works.  Unless you're thinking maybe this would help debugging issues, since we'd have some value in the DB at least describing what it is?

Reply:
> Unless you're thinking maybe this would help debugging issues, since we'd have some value in the DB at least describing what it is?

Yeah, that was the thought at least.  Though, since this will always just a `yaml` doc, I could be convinced that this isn't that hard to determine and there is possibly an unknown here with mucking with STI columns.

* * *

I will leave this call up to you.  It isn't hard for me to change it so it is more of what we see more value in:  easier debugging or more "true to intent"/stable code.

Reply:
Yeah, I'm thinking drop the resource_type bit given we can just look at the yaml and probably know immediately that it's a state var hash.

Reply:
Fixed.